api/src/task/apply/create_application_submission_task.py

```diff
@@ -387,6 +445,9 @@ def create_manifest_text(submission: SubmissionContainer) -> str:
         Attachments included in ZIP (total #)
         1. my-attachment.txt (size X)
         2. another-attachment.docx (size X)
+
+        XML Files included in ZIP (total #)
+        1. GrantApplication.xml (size X)
```

Comment:
I checked what the manifest looks like when it has an xml file. Looks like it adds a whole section like so:

```
Manifest for Grant Application # GRANT00838603

Grant Application XML file (total 1): 
 1. GrantApplication.xml. (size 13390 bytes)

Forms Included in Zip File(total 2):
 1. Form SFLLL_2_0-V2.0.pdf (size 20927 bytes)
 2. Form SF424_Short_3_0-V3.0.pdf (size 21985 bytes)

Attachments Included in Zip File (total 0):
```

Could we add that (or make a follow-up ticket to do that?)